### PR TITLE
Update acquire-and-configure-loading-server.md

### PR DESCRIPTION
**Pull request by AndersUP**
One typo fixed. 

More importantly, trying to address inconsistency between Supported Operating Systems and Performance + Security, where Supported OS is currently limited to WS2008 & 2012, whereas Performance and Security Notices both refer to "Windows Server 2012 and beyond". I have customers that ask as this doc is unclear. I would expect newer versions to be supported but hope this edit brings the question to those that know so the documentation can be more consistent.

### CODE REVIEW

**Comment by jborsecnik** [comment]
@charlesfeddersen, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/analytics-platform-system/acquire-and-configure-loading-server.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ The loading system can be designed as one or more loading servers that perform c
...
   
-## <a name="Step2"></a>Step 2: Acquire the sServer  
+## <a name="Step2"></a>Step 2: Acquire the Server  
 Now that you better understand your capacity requirements, you can plan the servers and networking components that you will need to purchase or provision. Incorporate the following list of requirements into your purchasing plan, and then purchase your server or provision an existing server.  
   
 ### <a name="R"></a>Software Requirements  
 Supported Operating Systems:  
   
--   Windows Server 2012 or Windows Server 2012 R2. These operating systems require the FDR network adapter.  
+-   Windows Server 2012, Windows Server 2012 R2 or beyond. These operating systems require the FDR network adapter.
```
Comment (WilliamDAssafMSFT) [inline docs/analytics-platform-system/acquire-and-configure-loading-server.md]:
```suggestion
-   Windows Server 2012, Windows Server 2012 R2, or later versions. These operating systems require the FDR network adapter.  

```

**Comment by WilliamDAssafMSFT** [comment]
@AndersUP Thanks for taking the time to create this pull request! Sorry for the delay in responding, this looks pretty clear. I've made a couple tweaks. This doc update is in flight.

**Comment by WilliamDAssafMSFT** [comment]
#assign:WilliamDAssafMSFT

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Pull request merged**